### PR TITLE
chore: throw errors if needed

### PR DESCRIPTION
**Pull request by KFlash**
This throws errors in the test runner if wrong or missing input

error

### CODE REVIEW

**Comment by KFlash** [comment]
@aladdin-add I used "pretty-error" package for this. Could you "extract" what's needed from that lib and remove the dependency?

**Comment by aladdin-add** [comment]
the linting was failed somehow.

File: test/runner/utils.ts [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import { readFile, statSync, readdirSync, writeFile, existsSync } from 'fs';
 
+const PrettyError = require('pretty-error');
```
Comment (aladdin-add) [inline test/runner/utils.ts:3]: I guess you copied it from somewhere, but do not use `require()` here for consistentcy. :)

Comment (KFlash) [reply test/runner/utils.ts:3]: I had to use 'require' because there are no ts types for 'pretty-error'

Comment (aladdin-add) [reply test/runner/utils.ts:3]: declare module 'pretty-error';
```

Comment (KFlash) [reply test/runner/utils.ts:3]: Can you take over this PR and merge it?

File: test/runner/utils.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import { readFile, statSync, readdirSync, writeFile, existsSync } from 'fs';
 
-const PrettyError = require('pretty-error');
+import PrettyError from 'pretty-error';
 
-export const snapshotsFolderName = '__snapshots__'
+export const snapshotsFolderName = '__snapshot__'
```
Comment (KFlash) [inline test/runner/utils.ts:5]: it's more than 1 snapshot in that folder so should add (s) behind

Comment (aladdin-add) [reply test/runner/utils.ts:5]: you need to rename the folder if you put it that way. ;)

will do it later.

**Pull request merged**